unit-10/javascrip/main.js

```diff
@@ -0,0 +1,104 @@
+// lesson 1
+function printMultiplicationTable(number) {
```

Comment:
```suggestion
function printMultiplicationTable (number) {
```
Nếu là khai báo 1 hàm thì thêm dấu cách trước `(`